ethcore/sync/src/api.rs

```diff
@@ -948,12 +946,12 @@ impl LightSyncProvider for LightSync {
 	fn peer_numbers(&self) -> PeerNumbers {
 		let (connected, active) = self.proto.peer_count();
 		let peers_range = self.num_peers_range();
-		debug_assert!(peers_range.end > peers_range.start);
+		debug_assert!(peers_range.end() > peers_range.start());
```

Comment:
Should it be `>=` here and below?

Reply:
Yeah, I think so otherwise this fail if `start == end` because end was exclusive before